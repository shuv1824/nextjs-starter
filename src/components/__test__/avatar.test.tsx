import React from "react"
import Avatar from "../avatar"
import TestRenderer from "react-test-renderer"
import { cleanup } from "@testing-library/react"

afterEach(cleanup);

describe('Avatar', () => {
    it('should render without throwing an error', () => {
        const testRenderer = TestRenderer.create(<Avatar />);
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
})
