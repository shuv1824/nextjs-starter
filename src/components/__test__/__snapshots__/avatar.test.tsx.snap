// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render without throwing an error 1`] = `
<div>
  <img
    alt="avatar image"
    src="/avatar.png"
  />
</div>
`;
